tailwind.config.js

```diff
@@ -1,10 +1,26 @@
 import defaultTheme from "tailwindcss/defaultTheme";
 
+const plugin = require("tailwindcss/plugin");
+
```

Comment:
```suggestion
import defaultTheme from "tailwindcss/defaultTheme";
import plugin from "tailwindcss/plugin";
```

일관성을 위해 import문을 사용하시면 좋을 것 같습니다!